### DISCUSSION TITLE
Space in path or filename on a multi_single selection

### DISCUSSION THREAD

**Discussion by CodyOakes** [moudey/Shell#447] [Q&A] [answered]
I would like to use [PDFtk](https://www.pdflabs.com/tools/pdftk-the-pdf-toolkit/) to merge selected pdfs into a single pdf, this menu item currently works so long as there aren't any white spaces in the file names or file paths of the selected files. 
 
`item(type='file' mode='multi_single' find='.pdf' title='Merge PDFs' image=\uE202 cmd='cmd' arg='/k pdftk @sel cat output MERGED_PDF.pdf' wait='true')`
(I have added the cmd as cmd and the to the args /k just so the output window would stay open after running the cmd) 

Given the following path with the following files:
![PathFiles](https://github.com/moudey/Shell/assets/15006927/62d169b0-2e70-493f-9500-36101e45b110)

It assumes additional files for each whitespace:
![Output1](https://github.com/moudey/Shell/assets/15006927/ab095af2-2214-4ae0-8258-413dd15eb3fd)

If I replace `@sel` with `"@sel"` then all the selected files are treated as a single file selection by PDFtk: 
![Output2](https://github.com/moudey/Shell/assets/15006927/d7683711-8430-4587-8dc4-27db4d322d6e)

Is there a way to add "" around each selection in the `@sel` reference? Is there another work around that I'm missing in the docs? 
Any help would be greatly appreciated. Thanks!

**Comment by RubicBG** [comment, marked as answer]
`sel(true)` -  https://nilesoft.org/docs/functions/sel

> Add a third parameter to the sel function to change the path separators. 
> It is possible to assign the quote to the first parameter through these options
> 0 = Unquote, 1 = Double quote, 2 = Single quote, Or use a string instead.
> 
> sel(0, ",", "/")
> sel(1, ",", '\\')
> sel(2, ",", "/")
> sel('\"', "\n", ">")

(source: https://discord.com/channels/1106387012707168318/1112126326665650176/1150156808829292615)

**Reply by CodyOakes** [reply]
Sweet! Worked like a charm, not sure how I missed that in the docs. Thank you for your help @RubicBG
